src/config.rs

```diff
@@ -59,34 +55,17 @@ pub(crate) const CONNECTION_TIMEOUT: u64 = 5;
 #[serde(deny_unknown_fields)]
 #[non_exhaustive]
 pub struct Config {
-    #[serde(default = "Slot::<Admin>::with_default")]
-    pub admin: Slot<Admin>,
     #[serde(default)]
     pub clusters: Slot<ClusterMap>,
```

Comment:
(non blocking) what are your thoughts on keeping static cluster config? (since it's proxy specific)

Reply:
Well clusters is one of the least proxy specific pieces of information because the control plane and relay create a cluster map for proxies to consume through xDS.

We do still have static configuration, the API just also handles being updated through other sources if provided, unless I'm missing the question?

Reply:
Ah yes - that is a good point - everyone needs clusters/endpoints, so it's not actually proxy specific 👍🏻

Question answered!